server.py

```diff
@@ -10,13 +13,60 @@
     allow_methods=['*'],
     allow_headers=['*'],
 )
+FILE='urls.db'
+
 @app.get("/")
 def root():
-    return({ "message": "Welcome to the URL Shortener" })
+    return({ "message": "hello world" })
+
```

Comment:
u dont really need this endpoint since u will never need to access this from the frontend